packages/react-vapor/src/components/table-hoc/utils/TableHOCUtils.ts

```diff
@@ -88,13 +97,26 @@ const getTablePredicates = (tableId: string, state: IReactVaporState): ITableHOC
 
 const getDatePickerId = (tableId: string) => `${tableId}-date-range`;
 
+const setFixedWidth = (width: number, isFixedWidth: boolean) => (isFixedWidth ? {style: {width, maxWidth: width}} : {});

```

Comment:
is the boolean only for old table without the width saved in the store? :thinking: